sonar-project.properties

```diff
@@ -0,0 +1,17 @@
+sonar.projectKey=paulgessinger_acts
+sonar.organization=paulgessinger
```

Comment:
Could we also have it run for ACTS as a organisation? I think this requires a new account?

Reply:
I overlooked this, but definitely!

Reply:
Changed now.